plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/navigation-extra-screen/sidebar-navigation-extra-screen.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from '@woocommerce/navigation';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { SidebarPatternScreen } from '../pattern-screen/sidebar-pattern-screen';
+
+import './style.scss';
+import { PATTERN_CATEGORIES } from '../pattern-screen/categories';
+
+const getPatternCategoryByPath = ( path: string ) => {
+	// Define the regex pattern to match everything after "/homepage/"
+	const regex = /\/homepage\/(.*)/;
```

Comment:
nit: If we want to be sure the path starts with `/homepage/`, we can validate with `path.startsWith( '/homepage/' )`.

Reply:
We can't do it in this way because:
- the URL is: `admin.php?page=wc-admin&customizing=true&path=/customize-store/assembler-hub/homepage/${category}`
- I want to get the `category` variable.